spec/integration/aws_cloudsearch_domain_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Chef::Resource::AwsCloudsearchDomain do
+  extend AWSSupport
+  when_the_chef_12_server "exists", organization: "foo", server_scope: :context do
+    with_aws "when connected to AWS" do
+      it "aws_cloudsearch_domain 'test-cloudsearch-domain' creates a cloudsearch domain" do
```

Comment:
RE, your speed concerns:

You should tag this test as `:super_slow` like https://github.com/chef/chef-provisioning-aws/blob/master/spec/integration/machine_spec.rb#L16